pkg/server/plugin/keymanager/awskms/awskms.go

```diff
@@ -610,3 +733,61 @@ func min(x, y time.Duration) time.Duration {
 	}
 	return y
 }
+
+func loadServerID(idPath string) (string, error) {
+	ok, err := serverIDExists(idPath)
```

Comment:
A more robust pattern than stat+readfile is to just readfile and if it fails use os.IsNotExist to determine if it failed because the file doesn't exist.